### DISCUSSION TITLE
Live Screener Feature Request

### DISCUSSION THREAD

**Discussion by Hisashi-Reyuga** [pranjal-joshi/Screeni-py#151] [Ideas]
When we track prices on 1 min or 5 mins time frame we need to re run commands frequently and manually to get latest prices, if the auto data update feature is implemented that would be a great feature & will help very much.

Thank you.

**Comment by pranjal-joshi** [comment]
Hi @Hisashi-Reyuga 
Considering the worst-case scenario, where the user is scanning for all 1500+ stocks, The time to download data and scan itself will be more than 1m/5m
That's why live screening is not implemented. It is ideal to implement live screening only for indices.

**Reply by Hisashi-Reyuga** [reply]
I understand, then maybe you can add another feature... price alerts that are sent on telegram or whatsapp (not for all stocks just for those that are in the list).

**Comment by Hisashi-Reyuga** [comment]
Right now we can only scan Broad Market Indices so can you also add some more features like, scanning through Sectoral Indices, Strategy Indices and Thematic Indices ? coz that might be more helpful if a certain sector is attractive.

**Comment by kavyas2912** [comment]
hi, I have raised issue.

live scanning of the two most traded indices viz NIFTY and BANKNIFTY was available as -5-EMA in version 1.x. This helped us in judging the intraday trend and trading.

Kindly make it, live scanning of these two indices,  available in v2.x too at the earliest.
